src/ui/src/configurables/base/tos-privacy.tsx

```diff
@@ -17,6 +17,8 @@
  */
 
 import * as React from 'react';
+export const termsUri = 'https://pixielabs.ai/terms';
+export const privacyUri = 'https://pixielabs.ai/privacy';
```

Comment:
Is it intentional that these are different from the ones in the footer (defined [below](https://github.com/pixie-io/pixie/blob/e0aa3e2d820927ac8834d36c134643c7ad748f59/src/ui/src/configurables/base/tos-privacy.tsx#L23-L26))? Is it because signing up for CC4P is outside of CNCF/LF?

Reply:
That's a question for @zasgar, I think. Note that we use the same Linux Foundation footers on `px.dev`.

Also, the only reason these first two are still pointing to `pixielabs.ai` is that we never bothered to put them up on `px.dev`, otherwise I'd say the base config should only be pointing to `px.dev`.